charms/promote-all-charms-and-bundles.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -eux
+
+CHARMS="
+  cs:~containers/easyrsa
+  cs:~containers/etcd
+  cs:~containers/flannel
+  cs:~containers/kubernetes-master
+  cs:~containers/kubernetes-worker
+  cs:~containers/kubeapi-load-balancer
+  cs:~containers/kubernetes-e2e
+  cs:~containers/kubernetes-core
+  cs:~containers/canonical-kubernetes
+"
+
+for charm in $CHARMS; do
+  CHARM="$charm" FROM_CHANNEL="$FROM_CHANNEL" TO_CHANNEL="$TO_CHANNEL" ./charms/promote-charm.sh
```

Comment:
And do we want this to be from channel to channel or do we want it to be revision # to channel?